src/components/PlanningTool.jsx

```diff
@@ -1803,6 +1822,18 @@ class PlanningTool extends Component {
                       onClick={this.redo}>
                 Redo
               </button>
+              <button
+                className="action-button"
+                onClick={this.showOverview}
+              >
+                Overview
+              </button>
+              <button
+                className="action-button"
+                onClick={this.showCurrentWeek}
+              >
+                Center
```

Comment:
"center" might be confusing, let's call it something like "Current Week" or something similar

Reply:
I named it based on the discussion. But I agree "Current week" is much better for user clarity.